hack/test/e2e-qemu.sh

```diff
@@ -44,6 +44,7 @@ function create_cluster {
     --memory 2048 \
     --cpus 2.0 \
     --cidr 172.20.1.0/24 \
+    --user-disk /var/lib/extra:100MB \
```

Comment:
can we add more partitions to test things in a more complete way?

Reply:
yep, working on that.